Valet/VALSecureEnclaveValet.m

```diff
@@ -262,27 +263,7 @@ - (BOOL)canAccessKeychain;
 
 - (BOOL)containsObjectForKey:(nonnull NSString *)key;
 {
-    NSDictionary *options = nil;
```

Comment:
This code has been moved down verbatim to `-containsObjectForKey:options:` so that the logic can be shared with `VALSinglePromptSecureEnclaveValet`